### DISCUSSION TITLE
Is there have some document for add new chain with rust?

### DISCUSSION THREAD

**Discussion by weixuefeng** [trustwallet/wallet-core#3701] [Q&A]
I had to add new chain (aleo)[https://github.com/AleoHQ/snarkVM], which algorithm is zero-knowledge, i had to add algorithm with rust into wallet-core, is there have some workflow for it?

**Comment by satoshiotomakan** [comment]
Hi @weixuefeng, the documentation is available here: https://developer.trustwallet.com/developer/wallet-core/newblockchain
